### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by ascarter** [xcoders/xcoders.github.com#1] [Team Posts]
Hey guys - I think I already know the answer to this but I got a dependbot alert on a vulnerability on the site attached to this org. I think everything has moved to https://xcoders.org correct? If so, I'll retire this site and simply forward the domain name seattlexcoders.org to xcoders.org.

Hope everyone is doing well.

**Comment by jsorge** [comment]
Our email is still on seattlexcoders.org, but the website is moved over to xcoders.org. Is that going to be an issue?

**Comment by ascarter** [comment]
Not at all. I'm just talking about the website which has the source here. But I believe you guys built a new up-to-date site on https://xcoders.org. I can just change the dns for the web site to point to that for seattlexcoders.org and can retire this website.

**Comment by ascarter** [comment]
@jsorge following up on the site (I know - very late). I just set the domain seattlexcoders.org to forward to xcoders.org so I can retire the website (and quit getting dependbot notices). I see in the DNS config that there are still MX records, etc to google. Is Google mail still being used? I'd have to dust off how to see that mail - I might have set it up to forward to my GMail a long time ago.

I'm going to keep the domain name (it's cheap enough) and let it be an alias to xcoders.org so people stay with the current stuff. Let me know if you need to change anything with the email. I use Hover for the DNS and domain management.

**Comment by jsorge** [comment]
Yep the forwarding is perfect. We’re on micro.blog now and it’s been great. Emails still all go to the seattlexcoders.org domain, but getting something set up to use the xcoders.org one might be good as a transition point too.

I honestly don’t have the bandwidth to take that part on at the moment though. I also use Hover to manage the domain and DNS and all that too.

**Comment by ascarter** [comment]
OK - I'll see if I can just forward web domain to xcoders.org and if not,
I'll change the existing site to be a static page that references
xcoders.org. The email config (MX records, etc) I won't touch at all.
